README.md

```diff
@@ -28,11 +27,26 @@ E or R, when already that color, resets it to black.
 There are also keybindings with Shift and Ctrl, which you can view in-app.
 Other neat things you can do:
 
-- Hit the save button to save to a PDF.
-  The export button exports to a JSON file, which you can later open and edit.
-- Hit Ctrl+V to paste images from the system clipboard.
-  You can also drag images to the board.
-- Right-clicking to bring up a context menu to change the style.
+- Hit the export button to save to a PDF.
+  The save button exports to a JSON file, which you can later load back in to qboard.
+- Use your browser's paste function (usually Ctrl+V or Cmd+V on Mac) to paste images from the system clipboard.
+  You can also drag images onto the board, or use the file picker on the left side of the screen (labeled `Open`).
+- Open your saved JSON files with the file picker to replace your current board with one from a file.
+  Alternatively, drag and drop your saved JSON files;
+  instead of overwriting your current board, this will insert the contents of the files into the board after the current page!
+- Right-click to bring up a context menu to change the style.
+- If you have a saved JSON file accessible via a URL, you can make a link that preloads the board with that file!
+Take any link to qboard, like <https://pihart.github.io/qboard/>, and append `?json=<your URL here>`, to get <https://pihart.github.io/qboard/?json=example.json>.
```

Comment:
nit: can we make the example cjquines.com/qboard ?

actual comment: can we make the example just work and then just link to the example without explanation?

Reply:
Absolutely yes to both

Reply:
Assuming you wanted the trailing slash, this is fixed in 73276e9